### DISCUSSION TITLE
how to find out my  kernel version

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1232] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm  on v129 and trying to downgrade but don't know how to check if my kernel version is 4 so I don't waste my time my board is dedde is that helps

**Comment by schoolexploitkid** [comment]
you can downgrade up to v125

**Comment by schoolexploitkid** [comment, marked as answer]
you can downgrade up to v125

**Reply by buddyted** [reply]
how do i
